templates/mdd/task3_qarecs/templates.yaml

```diff
@@ -0,0 +1,84 @@
+dataset: mdd
+subset: task3_qarecs
+templates:
+  1614890b-362c-4ee8-850d-841cf511d169: !Template
+    id: 1614890b-362c-4ee8-850d-841cf511d169
+    jinja: "Speaker 1: {{dialogue_turns.utterance[0]}}
+
+
+      Speaker 2: {{dialogue_turns.utterance[1]}}
+
+
+      Speaker 1: {{dialogue_turns.utterance[2]}}
+
+
+      Speaker 2:|||{{dialogue_turns.utterance[3]}}"
+    name: next_utterance_4
+    reference: Given the first three dialogues,  generate next utterance.
+  3e5a19e5-aa33-467a-bcd2-f84d99f32759: !Template
+    id: 3e5a19e5-aa33-467a-bcd2-f84d99f32759
+    jinja: "{{dialogue_turns.utterance[0]}}|||{{dialogue_turns.utterance[1]}}"
```

Comment:
Missing a task description right now
<img width="641" alt="Capture d’écran 2021-06-09 à 7 40 47 PM" src="https://user-images.githubusercontent.com/16107619/121442992-a8bb0300-c95a-11eb-9da8-188312bd0e99.png">

Reply:
In this, the task description is being provided in the dialogue itself - "I'm looking for a Jack Nicholon movie.", wdyt?

Reply:
👍gotcha